src/main/scala/com/wavesplatform/state2/diffs/TransactionDiffer.scala

```diff
@@ -24,7 +24,12 @@ object TransactionDiffer {
       t5 <- CommonValidation.disallowSendingGreaterThanBalance(s, settings, currentBlockTimestamp, t4)
       diff <- t5 match {
         case gtx: GenesisTransaction => GenesisTransactionDiff(currentBlockHeight)(gtx)
-        case ptx: PaymentTransaction => PaymentTransactionDiff(s, currentBlockHeight, settings, currentBlockTimestamp)(ptx)
+        case ptx: PaymentTransaction =>
```

Comment:
please move this `if` etc to `PaymentTransactionDiff` in order to keep things consistent